channels/rdpdr/client/rdpdr_main.c

```diff
@@ -921,6 +930,12 @@ static UINT rdpdr_process_receive(rdpdrPlugin* rdpdr, wStream* s)
 	UINT32 status;
 	UINT error;
 
+	if (!rdpdr || !s)
```

Comment:
IMHO this check should be removed and done outside in rdpdr_virtual_channel_client_thread.